### PR TITLE
use Polyfill

### PR DESCRIPTION
**Pull request by SimonCropp**
* adds Polyfill to Akka.csproj
 * uses a ModuleInitializer (which polyfill adds support for) for test config in Akka.API.Tests.csproj
 * updated Verify.Xunit to use the simpler `UseProjectRelativeDirectory` api

### CODE REVIEW

**Comment by SimonCropp** [comment]
need some help deciphering the build error. seems unrelated to my change?? https://dev.azure.com/dotnet/Akka.NET/_build/results?buildId=92267&view=logs&j=d83928d8-8bfe-51de-721e-9025f72ee732&t=21d77f07-2bef-56ac-91e9-8618e6456479

**Comment by Aaronontheweb** [comment]
> need some help deciphering the build error. seems unrelated to my change?? https://dev.azure.com/dotnet/Akka.NET/_build/results?buildId=92267&view=logs&j=d83928d8-8bfe-51de-721e-9025f72ee732&t=21d77f07-2bef-56ac-91e9-8618e6456479

We have some notoriously racy tests in our test suite - almost always they're caused by tests having tightly bounded execution times.

> Akka.Persistence.Query.InMemory.Tests.InMemoryEventsByTagSpec.ReadJournal_live_query_EventsByTag_should_find_new_events

This is one of them. No worries - you're in good shape. You can mark this PR as ready for review when you're set @SimonCropp

**Comment by Aaronontheweb** [comment]
The feature I'm adding to Akka.NET which depends on this is going to be brought into the main repo today I think - is this good for me to review?

**Comment by SimonCropp** [comment]
@Aaronontheweb yeah. ready for review.

File: src/core/Akka.API.Tests/Akka.API.Tests.csproj [hunk 1]
```diff
@@ -29,6 +29,7 @@
     <PackageReference Include="xunit" Version="$(XunitVersion)" />
     <PackageReference Include="xunit.runner.visualstudio" Version="$(XunitVersion)" />
     <PackageReference Include="PublicApiGenerator" Version="9.3.0" />
+    <PackageReference Include="Verify.Xunit" Version="19.14.0" />
```
Comment (Aaronontheweb) [inline src/core/Akka.API.Tests/Akka.API.Tests.csproj]: Duplicate reference - I'll remove it.

File: src/core/Akka.API.Tests/ModuleInit.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+//-----------------------------------------------------------------------
+
+using System.Runtime.CompilerServices;
+using VerifyTests;
+using VerifyXunit;
+
+public static class ModuleInit
+{
+    [ModuleInitializer]
+    public static void Init()
```
Comment (Aaronontheweb) [inline src/core/Akka.API.Tests/ModuleInit.cs:15]: LGTM

File: src/core/Akka/Akka.csproj [hunk 3]
```diff
@@ -14,6 +14,7 @@
   </ItemGroup>
 
   <ItemGroup>
+      <PackageReference Include="Polyfill" Version="1.21.0" PrivateAssets="all" />
```
Comment (Aaronontheweb) [inline src/core/Akka/Akka.csproj:17]: LGTM

**Review by Aaronontheweb — APPROVED** [review]
LGTM

**Comment by Aaronontheweb** [comment]
Looks great, thank you for this @SimonCropp

**Pull request merged**